testnets/holesky/README.md

```diff
@@ -0,0 +1,791 @@
+
+- `cb.docker-compose.yml`: which contains the full setup of the Commit-Boost services
+- `.cb.env`: with local env variables, including JWTs for modules
+- `target.json`: which enables dynamic discovery of services for metrics scraping via Prometheus
+
+**Running**
+
+The final step is to run the Commit-Boost services. This can be done with the following command:
+
+```bash
+commit-boost start --docker cb.docker-compose.yml --env .cb.env
+```
+
+This will run all modules in Docker containers.
+
+> [!IMPORTANT] > `bolt-boost` will be exposed at `pbs.port` (18551 by default, set with `BOLT_SIDECAR_BUILDER_PROXY_PORT`), and your beacon node MUST be configured
+> to point the `builder-api` to this port for Bolt to work.
+
+### Observability
+
+Commit-Boost comes with various observability tools, such as Prometheus,
+cadvisor, and Grafana. It also comes with some pre-built dashboards, which can
+be found in the `grafana` directory.
+
+To update these dashboards, run the following command:
+
+`bash ./update-grafana.sh `
+
+In this directory, you can also find a Bolt dashboard, which will be launched
+alongside the other dashboards.
+
+# Register your validators on-chain on the Bolt Registry
+
+Once you are successfully running the Bolt sidecar you need to register on-chain
+on the Bolt Registry to successfully receive preconfirmation requests from users
+and RPCs. This step is needed to prove that your commitments are economically
+backed.
+
+In order to do that you need some collateral in the form of whitelisted Liquid
+Staking Token (LST) that needs to be restaked in either the Symbiotic or
+EigenLayer protocol.
+
+Then you need to interact with two contracts on Ethereum Mainnet:
+`BoltValidators` and `BoltManager`. The former is used to register your
+active validators into the protocol, while the latter is used to manage to
+register as an operator into the system and integrate with the restaking
+protocols.
+
+## Validator Registration
+
+The [`BoltValidators`](./src/contracts/BoltValidators.sol) contract is the only point of entry for
+validators to signal their intent to participate in Bolt Protocol and authenticate with their BLS private key.
+
+The registration process includes the following steps:
+
+1. Validator signs a message with their BLS private key. This is required to prove that the
+   validator private key is under their control and that they are indeed its owner.
+2. Validator calls the `registerValidator` function providing:
+   1. Their BLS public key
+   2. The BLS signature of the registration message
+   3. The address of the authorized collateral provider
+   4. The address of the authorized operator
+
+Until the Pectra hard-fork will be activated, the contract will also expose a `registerValidatorUnsafe` function
+that will not check the BLS signature. This is gated by a feature flag that will be turned off post-Pectra and
+will allow us to test the registration flow in a controlled environment.
```

Comment:
I think we need some additional information here on how to register these validators. Can work on some scripts like last time.

Reply:
If that's okay with you I'll update this section once we have the scripts!